README.md

```diff
@@ -27,16 +27,10 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 ## Building
 
-`runc` only supports Linux. It must be built with Go version 1.21 or higher.
+`runc` only supports Linux. It must be built with Go version 1.22.4 or higher.
 
 ### Pre-Requisites
 
-#### Go
-
-NOTE: if building with Go 1.22.x, make sure to use 1.22.4 or a later version
-(see [issue #4233](https://github.com/opencontainers/runc/issues/4233) for
-more details).
-
```

Comment:
To other reviewers: this is no longer needed since PR #4407.

Reply:
Should i change 1.22.4 to 1.22 in my changes?

Reply:
IMHO just removing this is fine. I think @kolyshkin was just giving context to other reviewers on why removing that note seems fine.